exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/lateraljoin/TestLateralPlans.java

```diff
@@ -90,8 +90,50 @@ public void testLateralSqlPlainCol() throws Exception {
 
   @Test
   public void testLateralSqlStar() throws Exception {
-    String Sql = "select * from cp.`lateraljoin/nested-customer.json` t, unnest(t.orders) t2 limit 1";
-    test(Sql);
+    String Sql = "select * from cp.`lateraljoin/nested-customer.json` t, unnest(t.orders) t2 limit 0";
+
+    testBuilder()
+            .unOrdered()
+            .sqlQuery(Sql)
+            .baselineColumns("c_name", "c_id", "c_phone", "orders", "c_address", "o_id", "o_shop", "o_amount", "items")
```

Comment:
I think this set of columns is not right.  The UNNEST should produce only the top level column since that's what the  planner knows about.  De-constructing the underlying map in the data means that we will not be able to specify a column alias for the unnest output:  e.g   UNNEST(t.orders) as t2(o).

Reply:
The set of columns returned from UNNEST is another issue. And can be changed later if really needed. This fix is mainly about star case.